analyzer/src/main/kotlin/managers/Conan.kt

```diff
@@ -325,6 +330,28 @@ class Conan(
     private fun parsePackageField(node: JsonNode, workingDir: File, field: String): String =
         inspectField(node["display_name"].textValue(), workingDir, field)
 
+    /**
+     * Try to read the source artifact from the [conanStoragePath], if not possible return [RemoteArtifact.EMPTY].
+     */
+    private fun parseSourceArtifact(id: Identifier, conanStoragePath: File): RemoteArtifact {
+        val conanDataFile = conanStoragePath.resolve("${id.name}/${id.version}/_/_/export/conandata.yml")
+
+        return kotlin.runCatching {
+            val conanData = yamlMapper.readTree(conanDataFile)
+            val artifactEntry = conanData["sources"][id.version]
+
+            val url = artifactEntry["url"].let {
+                if (it.isTextual) it.textValueOrEmpty() else it.first().textValueOrEmpty()
+            }
+            val hash = artifactEntry["sha256"].textValueOrEmpty().takeIf { it.isNotBlank() }
+                ?.let { Hash(it, HashAlgorithm.SHA256) } ?: Hash.NONE
```

Comment:
Could be just

    val  hash = Hash.create(artifactEntry["sha256"].textValueOrEmpty())